docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -382,7 +382,7 @@ start](#kubestellar-start) command above.
 ### Fetch a log from a KubeStellar runtime container
 
 {%
-   include-markdown "../../../../core-helm-chart/README.md"
+   include-markdown "../../../../chart/README.md"
```

Comment:
This document is obsolete, updating it for this one change would be confusing.

Reply:
ok, change undone